cylc/flow/install_plugins/log_vc_info.py

```diff
@@ -0,0 +1,317 @@
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Record version control information to the workflow log directory on
+installation.
+
+If the workflow source directory is a supported repository/working copy
+(git or svn), information about the working copy will be saved in
+``<run-dir>/log/version/vcs.conf``.
+
+An example of this information for a git repo:
+
+.. code-block:: cylc
+
+   version control system = "git"
+   repository version = "2.8.0-dirty"
+   commit = "e5dc6573dd70cabd8f973d1535c17c29c026d553"
+   working copy root path = "~/cylc-src/my-workflow-git"
+   status = \"\"\"
```

Comment:
I don't think these quotes should be escaped here?

Reply:
Because the whole docstring is a triple quote string? Rendering fine: 

<img src="https://user-images.githubusercontent.com/61982285/116237991-c2acd780-a758-11eb-99ea-a640d18784c3.png" width="500" />